modules/storages/app/views/storages/admin/storages/show.html.erb

```diff
@@ -134,7 +134,7 @@ See COPYRIGHT and LICENSE files for more details.
       <div class="attributes-key-value--key"><%= t("storages.provider_types.#{@object.provider_type}") %> <%= t(:'storages.label_oauth_client_secret') %></div>
       <div class="attributes-key-value--value-container">
         <div class="attributes-key-value--value -text">
-          <span><%= short_secret(@object.oauth_client.client_secret) %></span>
+          <span>●●●●●●●●●●●●●●●●</span>
```

Comment:
@apfohl Is it necessary to remove the shortening of the Nextcloud OAuth client secret? Here, the value is not hashed in our DB so we technically can reveal some characters. I argue that it increases the UX for the user as she can still compare the first two and the last two letters.